server/package.json

```diff
@@ -17,6 +17,12 @@
     "cookie-parser": "~1.4.3",
     "dotenv": "^4.0.0",
     "express": "^4.15.2",
+    "eslint": "^3.19.0",
+    "eslint-config-airbnb": "^14.1.0",
+    "eslint-loader": "^1.7.1",
+    "eslint-plugin-import": "^2.2.0",
+    "eslint-plugin-jsx-a11y": "^4.0.0",
```

Comment:
Can't clean up these since they're dependencies for eslint-config-airbnb, which is the config we're extending for the linting.